cookbook/README.md

```diff
@@ -0,0 +1,96 @@
+# baragon-cookbook
+
+Installs Baragon (service or agent) on a node.  Baragon is the loadbalancer management service for [HubSpot's Singularity cluster management system](https://github.com/HubSpot/Singularity).
+
+## Supported Platforms
+
+Tested on an extensive range of platforms including Ubuntu 14.04 and nothing else.
+
+## Requirements
+
+### Zookeeper
+
+Baragon depends on Zookeeper to receive requests from Singularity and to relay requests from the service to the agent.  Using an existing Zookeeper cluster is totally fine if you happen to have one.
```

Comment:
a more accurate description of Baragon's ZK usage would be something like: `Baragon depends on ZooKeeper to store internal state, and perform coordination between multiple BaragonService instances if high availability is desired. Using an existing Zookeeper cluster is totally fine if you happen to have one. Separate Baragon installations can also share the same Zookeeper cluster, provided they use different zkNamespace values.`